index.html

```diff
@@ -0,0 +1,192 @@
+                    <li><a href="about.html" id="active">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Courses</a></li>
+                    <li><a href="#">Contact</a></li>
+                    <button type="submit" class="btn btn-default">
+                        <a href="#">New Courses!</a>
+                    </button>
+                </ul>
+                <i class="open fa fa-bars"></i>
+                <ul class="mobile-nav">
+                    <i class="close fa fa-times"></i>
+                    <li><a href="index.html" id="active">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Courses</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>
+        </nav>
+
+        <!-- ------ start of home section  -->
+        <section class="home">
+            <div class="container">
+                <div class="home-text">
+                    <h2>Welcome! to Evans code online school</h2>
+                    <h1>GROW YOUR CODING SKILLS TO ADVANCE YOUR CAREER PATH</h1>
+                    <p>Our aim to help you a establish a good fundation in software development. The journey
+                        of becoming a developer can not just be achieved by feeling good behind a complter. We make sure
+                        you gain much understanding of every basic lesson in coding.
+                    </p>
+                    <h3>Starting 05.06.23(Mon) ~ (Fri)</h3>
+                    <h6>@ 8am Opo-louker street, West Hills-Weija. Ghana</h6>
+                    <!-- <a href="#" class="hero-btn">Visit us</a> -->
+                </div>
+            </div>
+
+        </section>
+    </header>
+
+    <!-- start of course feature ------>
+    <course class="course">
+        <section class="course-feature">
+            <h2 class="course white">Course Category</h2>
+            <div class="container">
+                <div class="course-card flex">
+                    <div class="course-img">
+                        <img src="images/course/program_icon_01.png" alt="course-icon">
+                    </div>
+                    <h3 class="tuitor">software D.</h3>
+                    <p class="white">Get the opportunities to network with CC affiliates around the world, also after
+                        the
+                        summit.</p>
+                </div>
+                <div class="course-card flex">
+                    <div class="course-img">
+                        <img src="images/course/program_icon_02.png" alt="course-icon">
+                    </div>
+                    <h3 class="tuitor">Machine L</h3>
+                    <p class="white">Get the opportunities to network with CC affiliates around the world, also after
+                        the
+                        summit.</p>
+                </div>
```

Comment:
- [ ] As I mentioned above :point_up_2: please stick to the Behance design as much as possible but in the mobile version your `Course Category` is not the same as Behance design :+1: check below :point_down: comparison between your design and Behance Design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-05-27 00-03-29](https://github.com/evansnyamekye/Capstone-PR/assets/45226848/f9b040a0-3242-47a4-840c-0236868f767d)

**Behance Design**
![Screenshot from 2023-05-27 00-03-49](https://github.com/evansnyamekye/Capstone-PR/assets/45226848/f7290f4d-49cf-48f3-b813-164e08969592)

- [ ] If you check your design there are some spacing and margins problems kindly fix these issues as well :+1: